### ISSUE TITLE
Reconcilliation loop between GAPI of Kuma and KIC

### ISSUE THREAD

**Issue by jakubdyszkiewicz** [kumahq/kuma#9158]
### What happened?

Run KIC e2e test https://github.com/kumahq/kuma/pull/9157
See that Kong goes crazy, because Kuma overrides HTTPRoute status, then Kong, then Kuma etc.
I don't think we should put anything into status of HTTPRoute that are not for meant for Kuma GatewayClass.

**Comment by michaelbeaumont** [comment]
> I don't think we should put anything into status of HTTPRoute that are not for meant for Kuma GatewayClass.

Of course, but this is supposed to already be taken into account, it's part of the spec. There might be a bug handling the lists I suppose
